diff -ru site/inn.conf /usr/local/news/etc/inn.conf
--- site/inn.conf	2015-05-04 20:37:41.393424837 +0100
+++ /usr/local/news/etc/inn.conf	2015-05-04 20:15:22.414295431 +0100
@@ -21,7 +21,7 @@
 organization:                "A poorly-installed InterNetNews site"
 ovmethod:                    tradindexed
 hismethod:                   hisv6
-pathhost:                    deodand
+pathhost:                    test.anjou.terraraq.org.uk
 pathnews:                    /usr/local/news
 
 #runasuser:
@@ -32,7 +32,7 @@
 #domain:
 #innflags:
 mailcmd:                     /usr/local/news/bin/innmail
-#server:
+server:			     deodand.anjou.terraraq.org.uk
 
 # Feed Configuration
 
@@ -49,7 +49,7 @@
 #pathalias:
 #pathcluster:
 pgpverify:                   true
-port:                        119
+port:                        1119
 refusecybercancels:          false
 remembertrash:               true
 #sourceaddress:
@@ -80,8 +80,8 @@
 
 allownewnews:                true
 articlemmap:                 true
-clienttimeout:               1800
-initialtimeout:              10
+clienttimeout:               86400
+initialtimeout:              3600
 msgidcachesize:              16000
 nfsreader:                   false
 nfsreaderdelay:              60
@@ -111,6 +111,7 @@
 addinjectionpostinghost:     true
 checkincludedtext:           false
 #complaints:
+complaints:			rjk@greenend.org.uk
 #fromhost:
 localmaxartsize:             1000000
 #moderatormailer:
