diff -ru site/innfeed.conf /usr/local/news/etc/innfeed.conf
--- site/innfeed.conf	2015-05-04 20:37:41.861424956 +0100
+++ /usr/local/news/etc/innfeed.conf	2015-05-04 15:11:38.411832725 +0100
@@ -127,3 +127,7 @@
 #     deliver-authname, deliver-password, deliver-username,
 #     deliver-realm, deliver-rcpt-to, deliver-to-header.
 
+peer deodand.anjou.terraraq.org.uk {
+     ip-name: deodand.anjou.terraraq.org.uk
+}
+
diff -ru site/newsfeeds /usr/local/news/etc/newsfeeds
--- site/newsfeeds	2015-05-04 20:37:41.285424810 +0100
+++ /usr/local/news/etc/newsfeeds	2015-05-04 15:11:21.267852277 +0100
@@ -72,9 +72,9 @@
 ##  file is needed.
 
 # innfeed funnel master.
-#innfeed!\
-#	:!*\
-#	:Tc,Wnm*:/usr/local/news/bin/innfeed
+innfeed!\
+	:!*\
+	:Tc,Wnm*:/usr/local/news/bin/innfeed
 
 ##  Only uncomment this feed if both enableoverview and useoverchan are
 ##  set to true in inn.conf.  By default, innd will write out overview
@@ -143,3 +143,7 @@
 #capture!\
 #	:*/foo\
 #	:Tp,H2:/usr/local/bin/capture %s
+
+deodand.anjou.terraraq.org.uk\
+        :greenend.*/!local\
+        :Tm:innfeed!
diff -ru site/readers.conf /usr/local/news/etc/readers.conf
